features/step_definitions/frontend_steps.rb

```diff
@@ -9,3 +9,19 @@
   doc = Nokogiri::HTML(html)
   assert doc.css('#services-and-information')
 end
+
+Then /^I should see an input field for entering my postcode$/ do
+  @response.body.should have_field('postcode')
+end
+
+Then /^I should see the heading "(.*?)"$/ do |title|
+  @response.body.should have_css('h1', text: title)
```

Comment:
Hey, I've traced the method down to the matcher and it seems to run [`node.text.match(options[:text])`](https://github.com/jnicklas/capybara/blob/e7dc237ead86d1da826b2dfc92c6f2af8a21d7bb/lib/capybara/node/finders.rb#L182), using the `String` object's `match` method. Meaning the searched for text only has to be present in order for it to match, rather than be an exact match. So kind of like an include..?